hw/ip/otbn/doc/_index.md

```diff
@@ -417,6 +417,21 @@ Additionally, instructions and data stored in the instruction and data memory, r
 
 Refer to the [Data Integrity Protection]({{<relref "#design-details-data-integrity-protection">}}) section for details of how the data integrity protections are implemented.
 
+## Secure Wipe
+
+OTBN provides a mechanism to securely wipe all state it stores, including the instruction memory.
+
+This mechanism is a superset of other secure wipe mechanisms:
+- Data Memory Secure Wipe
+- Instruction Memory Secure Wipe
+- Internal State Secure Wipe
```

Comment:
Could these items link to the relevant sections? Also, are the capitals really necessary? It sounds a bit shouty to me.